backup/setup/migrations.json

```diff
@@ -95,12 +84,12 @@
 },
 {
 	"_id": {
-		"$oid": "5d4858fc8f66f50964686467"
```

Comment:
There are a lot of changes in this file. are these intentional? It seems to be the order of the migration changed?